Rakefile

```diff
@@ -1 +1,95 @@
-require "bundler/gem_tasks"
+require 'bundler/gem_tasks'
+require 'rake/clean'
+require 'cucumber/rake/task'
+require 'mechanize'
+require 'fileutils'
+require 'yaml'
+
+CDK_DOWNLOAD_URL='https://access.redhat.com/downloads/content/293/ver=2/rhel---7/2.0.0/x86_64/product-software'
```

Comment:
Probably better to extract these links in separate file says `constants` in order to reduce `Rakefile` size.